redhat-appstudio/service-provider-integration-operator
# Configurable lifetime per binding

### What does this PR do?
Allows to set configurable lifetime per every given binding or `-1` for infinite lifetime.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://issues.redhat.com/browse/SVPI-278

### How to test this PR?

- Create a binding with relatively low `spec.lifetime`, like 70 sec, wait for it do disappear after this time.
- Configure operator to have low defaut timeout, like 20 sec, and create binding with `-1` as a `spec.lifetime`, it should not be deleted after default TTL passing.
